serverless-cljs-plugin/serverless_lumo/build.cljs

```diff
@@ -85,7 +85,14 @@
 (defn build!
   "Build a project."
   [opts cljs-lambda-opts]
-  (let [compiler   (:compiler cljs-lambda-opts)
+  (let [compiler   (merge (when (:warning-exit opts)
```

Comment:
Please could you lift the warning handler into a top-level named function, so that this can be:

```
(merge 
  (when (:warning-exit opts)
   {:warning-handlers [exit-on-warning]})
 (:compiler cljs-lambda-opts))
```

Or similar.

Reply:
Sure I can

Reply:
Done